vmss/linux/variables.tf

```diff
@@ -4,11 +4,25 @@ variable "environment" {}
 variable "application_name" {}
 variable "subnet_id" {}
 variable "ssh_public_key" {}
+variable "admin_password" {}
+variable "cluster_ip" {}
 
 variable "admin_username" {
   default = "azure"
 }
 
+variable "vmss_size" {
```

Comment:
Could this variable be changed to match the attribute name of "name" for the SKU hash? It seems like some confusion could occur around the vmss_size vs vmss_name, unless size is more appropriate, but I feel like it could be mistaken for something else pretty easily.

Reply:
In the doc the name is the size.  From a supportability perspective I think size is a better more intuitive name.  Just name wouldn't make a lot of sense to me when defining size, but you're call.